_includes/legacy/sections/calendar-contacts-sync.html

```diff
@@ -55,6 +55,11 @@ <h3>Worth Mentioning</h3>
 <ul>
   <li>
     <a href="https://fruux.com/">fruux</a> - A unified contacts/calendaring system that works across platforms and devices.
+    {% include badge.html
+    color="warning"
+    icon="fas fa-exclamation-triangle"
+    tooltip="The source programming code for Fruux is not available. Fruux is a proprietary application."
```

Comment:
```suggestion
    tooltip="The source code for Fruux is not available. Fruux is a proprietary application."
```

The word "programming" in the middle here looks a bit strange for me